app/konnect/mesh-manager/migrate-zone.md

```diff
@@ -0,0 +1,100 @@
+   meshproxypatches meshratelimits meshretries meshtcproutes meshtimeouts meshtraces meshtrafficpermissions
+   proxytemplates rate-limits retries timeouts traffic-logs traffic-permissions traffic-routes traffic-traces
+   virtual-outbounds"
+    
+   meshes=$(kumactl get meshes -oyaml | yq '.items[].name')
+   
+   # For each service mesh we produce a single file ${mesh}.yaml that contains all the policies of the mesh including 
+   # the mesh resource itself
+   
+   for mesh in ${meshes}; do
+     kumactl get mesh ${mesh} -oyaml | yq '(del(.creationTime,.modificationTime))' > "${outdir}/${mesh}.yaml"
+     echo "---" >> "${outdir}/${mesh}.yaml"
+     
+     for type in ${types}; do
+       kumactl get ${type} --mesh ${mesh} -oyaml | yq '.items[] |= (del(.creationTime,.modificationTime)) | .items[] | split_doc' | grep ^ >> "${outdir}/${mesh}.yaml" && echo "---" >> "${outdir}/${mesh}.yaml"
+     done
+   done
+    ```
+
+1. If mTLS is enabled on the mesh and the backend type is `builtin` or `provided`, then copy the mesh secrets:
+    * **Builtin:** Copy secrets named `{mesh_name}.ca-builtin-cert-{backend_name}` and `{mesh_name}.ca-builtin-key-{backend_name}`. For more information, see [storage of secrets](/mesh/{{page.kong_version}}/policies/mutual-tls/#storage-of-secrets).
+    * **Provided:** Copy secrets specified in the mesh resource. For more information, see [usage of "provided" CA](/mesh/{{page.kong_version}}/policies/mutual-tls/#usage-of-provided-ca).
+
+1. Switch the active `kumactl` control plane to {{site.konnect_short_name}}:
+  ```bash
+  kumactl config control-planes switch --name {config}
+  ```
+
+1. Apply resources fetched from the self-managed global control plane in steps 2 and 3: 
+  ```bash
+  kumactl apply -f {file_name}
+  ```
+
+## Connect the zone control plane to {{site.konnect_short_name}}
+
+1. [Create a new zone in {{site.konnect_short_name}}](/konnect/mesh-manager/service-mesh/#create-a-zone-in-the-global-control-plane). The name of the new zone should match the zone from {{site.mesh_product_name}} that we're connecting to. 
+  Be sure to override your existing zone control plane `values.yaml` configuration with the values provided in the {{site.konnect_short_name}} UI wizard.
+
+1. If [KDS TLS](/mesh/latest/production/secure-deployment/certificates/#control-plane-to-control-plane-multizone) is enabled with self-signed certificates, set `kuma.controlPlane.tls.kdsZoneClient.secretName=""` so the zone control plane can use public certificates to authenticate with {{site.konnect_short_name}}:
+  ```bash
+  kumactl install control-plane \
+  --set "kuma.controlPlane.tls.kdsZoneClient.secretName=kds-ca-certs" \
+  | kubectl apply -f -
+  ```
+
+1. Restart zone control plane with the new values. {{site.konnect_short_name}} will automatically start looking for the zone. Once {{site.konnect_short_name}} finds the zone, it will display it.
```

Comment:
@lobkovilya Could we add the command to restart the control plane here? I tried to look it up in our docs but couldn't find it.

Reply:
I just checked, in GUI for Kubernetes we already provide the command that restarts CP if it already exists (`helm upgrade --install`):
<img width="871" alt="image" src="https://github.com/Kong/docs.konghq.com/assets/12110261/ad395678-c1d0-4aed-909c-39ba90977da7">

For Universal, it doesn't make sense to provide the command, because there are too many ways customers are running Zone CP, I don't think it is worth trying to cover all the cases.